src/koans/10_runtime_polymorphism.clj

```diff
@@ -10,19 +10,19 @@
                    "!")))
 
 (defmulti diet (fn [x] (:eater x)))
-(defmethod diet :herbivore [a] __)
-(defmethod diet :carnivore [a] __)
-(defmethod diet :default [a] __)
+(defmethod diet :herbivore [a] (str (:name a), " eats ", "veggies."))
```

Comment:
Convention is to not use whitespaces but It's not restricted by the compiler.
Also you would join last two strings into one string like: `(str (:name a) " eats veggies."))`